apps/bytebase/3.1.2/docker-compose.yml

```diff
@@ -1,6 +1,6 @@
 services:
   bytebase:
-    image: bytebase/bytebase:3.1.2
+    image: bytebase/bytebase:3.2.0
     container_name: ${CONTAINER_NAME}
     restart: always
     networks:
```

Comment:
The changes between the two versions of the `services.bytebase` section appear to be related to an update to the Docker image. Specifically, the version number has been changed from `3.1.2` to `3.2.0`. This is a normal operation when updating software or plugins to receive new features or bug fixes.

There are no issues with logic or syntax in this change, and it does not require optimization since only one property (`image`) was updated. However, ensure that if you're deploying Bytebase on a CI/CD platform like Kubernetes, Helm, or Ansible, you also need to update any configurations or annotations associated with this service name.

If there's another aspect of the configuration that needs review, feel free to let me know!